.circleci/config.yml

```diff
@@ -211,14 +228,15 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v.*/
-      - publish-latest:
+      - publish-docs:
+          context: aws
           requires:
             - setup
           filters:
             branches:
-              only: master
+              only: next
             tags:
-              only: /v[0-9]+\.[0-9]+\.[0-9]+$/
+              only: /v[0-9]+\.[0-9]+\.[0-9]+-next.*/
```

Comment:
Docs aren't necessary tied to version tags. We may wish to continually update docs without publishing a new version. Unfortunately, CircleCI doesnt have a good way to manually trigger workflows.

Optional: Consider triggering this workflow on every commit on the our release branches but have approval hold so we can dictate when to push a new version of docs.

Reply:
I also strongly encourage this change so you can actually try this change out without publishing `-next.999`. I get 5 emails every publish. Thanks.